client/js/stores/snippet-store.js

```diff
@@ -11,6 +11,7 @@ class SnippetStore {
     this.bindActions(RatingActions);
     this.state = {
       snippets: [],
+      currentUserSnippets: [],
```

Comment:
ratings and avg stay the same for both collections, only two different views depend on different collections of snippets (snippets and currentUserSnippets)